packages/block-library/src/post-author/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-author` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-author` block on the server.
+ *
+ * @return string Returns the filtered post author for the current post wrapped inside "h6" tags.
+ */
+function render_block_core_post_author() {
+	$post = gutenberg_get_post_from_context();
+	if ( ! $post ) {
+		return '';
+	}
+	return '<address>' . __( 'By ' ) . get_the_author() . '</address>';
```

Comment:
Should this us sprintf?

Reply:
What would be the benefit?

Reply:
In some languages, there might be no need for "by " or the author should come first. Ideally concatenation of translated strings and non translated ones should be avoided as much as possible because the sentence format changes from language to another.

Reply:
Oh, right. I totally forgot about the translation here.

Thanks for catching it.

Fixed!